packages/cli/generation/ir-migrations/src/migrations/v61-to-v60/migrateFromV61ToV60.ts

```diff
@@ -0,0 +1,96 @@
+        [GeneratorName.PYTHON_PYDANTIC]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.PYTHON_SDK]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.STOPLIGHT]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.POSTMAN]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.GO_FIBER]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.GO_MODEL]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.GO_SDK]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.RUBY_MODEL]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.RUBY_SDK]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.CSHARP_MODEL]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.CSHARP_SDK]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.SWIFT_MODEL]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.SWIFT_SDK]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.PHP_MODEL]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.PHP_SDK]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.RUST_MODEL]: GeneratorWasNeverUpdatedToConsumeNewIR,
+        [GeneratorName.RUST_SDK]: GeneratorWasNeverUpdatedToConsumeNewIR
+    },
+    jsonifyEarlierVersion: (ir) =>
+        IrSerialization.V60.IntermediateRepresentation.jsonOrThrow(ir, {
+            unrecognizedObjectKeys: "strip",
+            skipValidation: true
+        }),
+    migrateBackwards: (
+        v61: IrVersions.V61.IntermediateRepresentation
+    ): IrVersions.V60.ir.IntermediateRepresentation => {
+        return {
+            ...v61,
+            dynamic: v61.dynamic != null ? convertDynamic(v61.dynamic) : undefined
+        } as IrVersions.V60.ir.IntermediateRepresentation;
+    }
+};
+
+function convertDynamic(
+    dynamic: IrVersions.V61.dynamic.DynamicIntermediateRepresentation
+): IrVersions.V60.dynamic.DynamicIntermediateRepresentation {
+    return {
+        ...dynamic,
+        endpoints: convertDynamicEndpoints(dynamic.endpoints)
+    };
+}
+
+function convertDynamicEndpoints(
+    endpoints: Record<IrVersions.V61.dynamic.EndpointId, IrVersions.V61.dynamic.Endpoint>
+): Record<IrVersions.V60.dynamic.EndpointId, IrVersions.V60.dynamic.Endpoint> {
+    return Object.fromEntries(
+        Object.entries(endpoints)
+            .map(([key, endpoint]) => [key, convertDynamicEndpoint(endpoint)])
+            .filter(([_, endpoint]) => endpoint != null)
+    );
+}
+
+function convertDynamicEndpoint(
+    endpoint: IrVersions.V61.dynamic.Endpoint
+): IrVersions.V60.dynamic.Endpoint | undefined {
+    return {
+        ...endpoint,
+        response: convertDynamicEndpointResponse(endpoint.response)
+    };
+}
+
+function convertDynamicEndpointResponse(response: IrVersions.V61.dynamic.Response): IrVersions.V60.dynamic.Response {
+    return IrVersions.V61.dynamic.Response._visit<IrVersions.V60.dynamic.Response>(response, {
```

Comment:
This works! For simplicity, could also just do a single IrVersions.V60.dynamic.Response.json line :)